keyboards/keyboardio/atreus/rules.mk

```diff
@@ -0,0 +1,33 @@
+# MCU name
+MCU = atmega32u4
+
+# Bootloader selection
+#   Teensy       halfkay
+#   Pro Micro    caterina
+#   Atmel DFU    atmel-dfu
+#   LUFA DFU     lufa-dfu
+#   QMK DFU      qmk-dfu
+#   ATmega32A    bootloadHID
+#   ATmega328P   USBasp
+BOOTLOADER = caterina
```

Comment:
This line makes me sad. :cry:

Reply:
![](https://ksr-ugc.imgix.net/assets/028/406/015/936a9b0789681eef1cf3cdd6562dc611_original.jpg?ixlib=rb-2.1.0&w=680&fit=max&v=1583982518&auto=format&frame=1&q=92&s=0077f4b52d4890e99ab91640eb62d7f1)

Most likely this board just has atmel-dfu on it, which would make sense with @technomancy's comment below.